src/mlpack/methods/range_search/rs_model_impl.hpp

```diff
@@ -59,6 +59,23 @@ inline RSModel::RSModel(RSModel&& other) :
   other.rSearch = decltype(other.rSearch)();
 }
 
+inline bool RSModel::operator==(RSModel other)
+{
+  if ( (this->treeType == other.treeType) && (this->leafSize == other.leafSize)
```

Comment:
This doesn't conform with the style guide, it should be `if ((this->treeType ...` (and the same with the closing parentheses).